packages/plugins/i18n/server/constants/iso-locales.json

```diff
@@ -507,10 +611,54 @@
     "code":"en-IE",
     "name":"English (Ireland) (en-IE)"
   },
+  {
+    "code":"en-IM",
+    "name":"English (Isle of Man) (en-IM)"
+  },
+  {
+    "code":"en-IL",
+    "name":"English (Israel) (en-IL)"
+  },
   {
     "code":"en-JM",
     "name":"English (Jamaica) (en-JM)"
   },
+  {
+    "code":"en-JE",
+    "name":"English (Jersey) (en-JE)"
+  },
+  {
+    "code":"en-KE",
+    "name":"English (Kenya) (en-KE)"
+  },
+  {
+    "code":"en-KI",
+    "name":"English (Kiribati) (en-KI)"
+  },
+  {
+    "code":"en-LS",
+    "name":"English (Lesotho) (en-LS)"
+  },
+  {
+    "code":"en-LR",
+    "name":"English (Liberia) (en-LR)"
+  },
+  {
+    "code":"en-MO",
+    "name":"English (Macau SAR China) (en-MO)"
```

Comment:
in [this document](https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry) I found that `MO` region tag corresponds to **Macao** (not Macau)